build.mill

```diff
@@ -1,4 +1,4 @@
-//| mill-version: 1.0.3
+//| mill-version: 1.0.6-jvm
```

Comment:
Giving up, I see. 😉 This makes sense to me, though.

Reply:
I'm not happy about it lol but alas